scripts/pylib/twister/twisterlib/environment.py

```diff
@@ -313,6 +313,25 @@ def add_parse_arguments(parser = None):
                              "Default to html. "
                              "Valid options are html, xml, csv, txt, coveralls, sonarqube, lcov.")
 
+    parser.add_argument("--disable-coverage-aggregation", action="store_true", default=False,
```

Comment:
would this be better to group the new added options to a cov_opt = parser.add_argument_group()

Reply:
done